### ISSUE TITLE
Tried to create a gutter with a name that is already in use.

### ISSUE THREAD

**Issue by NeilTruick** [wallabyjs/atom-wallaby#18]
[Enter steps to reproduce below:]

1. Open Atom.
2. Open Wallaby configuration panel (Ctrl-1).
3. Press the Start button.

**Atom Version**: 1.0.19
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [atom-wallaby](https://github.com/wallabyjs/atom-wallaby) package, v1.0.6

### Stack Trace

Uncaught Error: Tried to create a gutter with a name that is already in use.

```
At C:\Users\ntruick\AppData\Local\atom\app-1.0.19\resources\app.asar\src\gutter-container.js:36

Error: Tried to create a gutter with a name that is already in use.
  at GutterContainer.module.exports.GutterContainer.addGutter (C:\Users\ntruick\AppData\Local\atom\app-1.0.19\resources\app.asar\src\gutter-container.js:36:15)
  at TextEditor.module.exports.TextEditor.addGutter (C:\Users\ntruick\AppData\Local\atom\app-1.0.19\resources\app.asar\src\text-editor.js:2613:35)
  at b.g.value (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:12963)
  at C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:6387
  at arrayEach (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby\node_modules\lodash\index.js:1286:13)
  at Function.forEach (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby\node_modules\lodash\index.js:5700:11)
  at b.<anonymous> (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:6329)
  at emitTwo (events.js:87:13)
  at b.emit (events.js:172:7)
  at EventEmitter.<anonymous> (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:10009)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:169:7)
  at EventEmitter._.extend._filesToTrack (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby\client.js:14:3286)
  at ChildProcess.<anonymous> (C:\Users\ntruick\.atom\packages\atom-wallaby\wallaby\client.js:14:1179)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:172:7)
  at handleMessage (internal/child_process.js:632:10)
  at Pipe.channel.onread (internal/child_process.js:421:11)

```

### Commands

```
     -5:02.8.0 core:move-right (atom-text-editor.editor.is-focused.autocomplete-active)
     -5:02.3.0 core:move-down (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "packagesWithKeymapsDisabled": [
      "change-case"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.28.14
atom-jade, v0.3.0
atom-wallaby, v1.0.6
change-case, v0.6.0
jshint, v1.8.0

# Dev
No dev packages
```

**Comment by ArtemGovorov** [comment]
Does Atom restart help or the issue shows up even after the restart?

**Comment by NeilTruick** [comment]
It happens even after the restart.

**Comment by ArtemGovorov** [comment]
Are there any files opened when you start wallaby? If so, try closing them all before the start and then try starting wallaby again.

**Comment by ArtemGovorov** [comment]
One more thing: could you please try re-installing the package to see if the error is there after the re-install as well?

**Comment by RichardSlater** [comment]
Restarting with no files opened fixed the issue for me:

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.6.0-beta8
**System**: Microsoft Windows 10 Home Insider Preview
**Thrown From**: [atom-wallaby](https://github.com/wallabyjs/atom-wallaby) package, v1.0.7


### Stack Trace

Uncaught Error: Tried to create a gutter with a name that is already in use.

```
At C:\Users\richa\AppData\Local\atom\app-1.6.0-beta8\resources\app.asar\src\gutter-container.js:36

Error: Tried to create a gutter with a name that is already in use.
    at GutterContainer.module.exports.GutterContainer.addGutter (C:\Users\richa\AppData\Local\atom\app-1.6.0-beta8\resources\app.asar\src\gutter-container.js:36:15)
    at TextEditor.module.exports.TextEditor.addGutter (C:\Users\richa\AppData\Local\atom\app-1.6.0-beta8\resources\app.asar\src\text-editor.js:2700:35)
    at b.g.value (C:\Users\richa\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:14079)
    at C:\Users\richa\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:6830
    at arrayEach (C:\Users\richa\.atom\packages\atom-wallaby\wallaby\node_modules\lodash\index.js:1286:13)
    at Function.forEach (C:\Users\richa\.atom\packages\atom-wallaby\wallaby\node_modules\lodash\index.js:5700:11)
    at b.<anonymous> (C:\Users\richa\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:6772)
    at emitTwo (events.js:87:13)
    at b.emit (events.js:172:7)
    at EventEmitter.<anonymous> (C:\Users\richa\.atom\packages\atom-wallaby\wallaby-atom\index.js:14:10908)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
    at EventEmitter._.extend._filesToTrack (C:\Users\richa\.atom\packages\atom-wallaby\wallaby\client.js:14:3855)
    at ChildProcess.<anonymous> (C:\Users\richa\.atom\packages\atom-wallaby\wallaby\client.js:14:1289)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at handleMessage (internal/child_process.js:685:10)
    at Pipe.channel.onread (internal/child_process.js:440:11)
```

### Commands

```
  3x -3:06.5.0 core:select-right (input.hidden-input)
  2x -3:05.4.0 core:backspace (input.hidden-input)
  3x -2:48.1.0 core:move-left (input.hidden-input)
     -2:47.5.0 editor:select-to-beginning-of-word (input.hidden-input)
     -2:46.2.0 editor:move-to-end-of-word (input.hidden-input)
     -2:45.7.0 editor:select-to-end-of-word (input.hidden-input)
     -2:45.1.0 core:backspace (input.hidden-input)
  4x -2:44.6.0 core:move-down (input.hidden-input)
     -2:43.2.0 core:move-left (input.hidden-input)
     -2:43 editor:select-to-beginning-of-word (input.hidden-input)
  2x -2:42.1.0 core:move-right (input.hidden-input)
     -2:41.6.0 editor:select-to-end-of-word (input.hidden-input)
     -2:41.1.0 core:backspace (input.hidden-input)
  2x -2:40.6.0 core:save (input.hidden-input)
     -2:12.2.0 core:delete (input.hidden-input)
  3x -2:11.1.0 core:save (input.hidden-input)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "autoHideMenuBar": true,
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-html-preview, v0.1.17
atom-svg-icon-snippets, v0.2.0
atom-wallaby, v1.0.7
color-picker, v2.1.1
file-icons, v1.6.18
jslint, v1.5.1
language-latex, v0.6.1
language-powershell, v2.1.0
language-svg, v0.9.0
linter, v1.11.3
linter-chktex, v1.2.5
minimap, v4.20.0
monokai, v0.18.0
node-debugger, vundefined
npm, v0.1.0
pdf-view, v0.50.0
pigments, vundefined
pretty-json, v0.4.1
project-manager, v2.7.6
svg-preview, v0.7.4
sync-settings, v0.7.2
vagrant, v0.1.0
xml-tools, v0.2.1

# Dev
No dev packages
```

**Comment by ArtemGovorov** [comment]
> Restarting with no files opened fixed the issue for me

Thanks for the note!
The issue should be fixed in Atom plugin v1.0.22. It's a separately updatable/versioned package part and will be updated automatically. So either wait a few minutes when it updates after you (re)start wallaby.js next time or you may [force the core update](https://github.com/wallabyjs/public/issues/413#issuecomment-173174486).
